apps/challenges/aws_utils.py

```diff
@@ -20,8 +20,9 @@
 from base.utils import get_boto3_client, send_email
 from evalai.celery import app
 
-logger = logging.getLogger(__name__)
+# from .models import ChallengeEvaluationCluster,Challenge
```

Comment:
@burnerlee can you uncomment this and move all imports from methods `setup_eks_cluster`, `create_subnets_and_arns` `create_cluster`, etc here.